engines/ospray/OSPRayModel.cpp

```diff
@@ -43,22 +48,52 @@ OSPData allocateVectorData(const std::vector<VecT>& vec,
     return ospNewData(totBytes / ospray::sizeOf(ospType), ospType, vec.data(),
                       memoryManagementFlags);
 }
+
+double interpolatedOpacity(const Vector2ds& controlPoints, const double x)
```

Comment:
```suggestion
double interpolatedOpacity(const Vector2ds& controlPoints, const double x) const
```

Reply:
free function, const doesn't apply

Reply:
This is assuming control points are sorted by x, is that guaranteed by the caller? I haven't seen anything sorting the values received from json for the moment.

Reply:
added the sorting, good catch!